samples/boards/intel_s1000_crb/dmic/dmic.rst

```diff
@@ -0,0 +1,46 @@
+.. _dmic_app-sample:
+
+Intel® S1000 Digital Microphone Sample Application
+##################################################
+
+Overview
+********
+
+A sample implementation of an application that uses the digital microphone
+driver and receives interleaved audio samples from a microphone array.
+
+This app configures the digital microphone driver, starts the driver,
+receives a number of frames and then stops the driver. This process is
+repeated a few times.
+
+Requirements
+************
+
+This application uses an Intel® S1000 Customer Reference Board (CRB)
```

Comment:
Is there a board available to the public they can use to run this sample?

Reply:
Available only under NDA